Corzbank.Web/ClientApp/src/app/header/wallet/wallet.component.ts

```diff
@@ -162,12 +229,87 @@ export class WalletComponent implements OnInit {
 			});
 	}
 
+	openCardMenu() {
+		this.hideAllMenus();
+		this.cardDataIsDisplayed = false;
+		this.openCardMenuIsDisplayed = !this.openCardMenuIsDisplayed;
+	}
+
+	openDepositMenu() {
+		this.hideAllMenus();
+		this.cardDataIsDisplayed = false;
+		this.openDepositMenuIsDisplayed = !this.openDepositMenuIsDisplayed;
+		this.cardForDeposit = this.cards[0];
+	}
+
+	createCard() {
+		const cardForCreatingForm = this.createCardForm.value;
+		cardForCreatingForm.userId = this.currentUserId;
+		this.cardService.createCard(this.createCardForm.value).subscribe(
+			(card) => {
+				console.log(card);
+				if (card === null) {
+					this.notificationService.showErrorNotification(
+						"This type of card already exists",
+						""
+					);
+				} else {
+					this.cards.push(card);
+					this.notificationService.showSuccessfulNotification(
+						"Card was successfuly created",
+						""
+					);
+				}
+			},
+			(error) => {
+				this.notificationService.showErrorNotification("Error", "");
+			}
+		);
+		this.createCardForm.reset();
+	}
+
+	increment() {
+		if (this.durationOfDeposit < 24) this.durationOfDeposit += 1;
```

Comment:
this.durationOfDeposit +=1 === this.durationOfDeposit++;
this.durationOfDeposit -=1 === this.durationOfDeposit--;

Reply:
resolved